lib/src/showcase.dart

```diff
@@ -35,49 +35,162 @@ import 'tooltip_widget.dart';
 
 class Showcase extends StatefulWidget {
   @override
+
+  /// A key that is unique across the entire app.
   final GlobalKey key;
 
+  /// Target widget that will be showcased or highlighted
   final Widget child;
+
+  /// Represents subject line of target widget
   final String? title;
-  final TextAlign? titleAlignment;
+
+  /// Title alignment with in tooltip widget
+  ///
+  /// Defaults to [TextAlign.start]
+  final TextAlign titleAlignment;
+
+  /// Represents summary description of target widget
   final String? description;
-  final ShapeBorder? shapeBorder;
-  final BorderRadius? radius;
+
+  /// ShapeBorder of the highlighted box when target widget will be showcased.
+  ///
+  /// Note: If [targetBorderRadius] is specified, this parameter is ignored.
```

Comment:
This should be,

If [targetBorderRadius] is specified, this parameter will be ignored.